sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -464,7 +464,8 @@ def input_elements(self,
             return
           if self._exc_info:
             t, v, tb = self._exc_info
-            raise t(v).with_traceback(tb)
+            if t and v and tb:
```

Comment:
One option to simplify this could be to initialize _exc_info differently:
```
self._exc_info = (None, None, None)  # type: OptExcInfo

... 

if t: 
  ...
```
This way we omit the `if self._exc_info` condition, and mypy passes.

Reply:
SGTM, fixed at 9b76f4a